ocaml/src/cli_bench_common.ml

```diff
@@ -45,6 +45,13 @@ let prefix default =
        & info ["prefix"] ~docv:"prefix" ~doc
   )
 
+let process_prefix default =
+  let doc = "$(docv) to keep multiple bench processes out of each other's way" in
```

Comment:
If I understand the code correctly this argument is about keeping the log output of the different processes out of each other's way... At first I thought it was about the keys used on the asd (for which there is of course already another option).
Maybe make that a bit more clear in both the description and the variable name as it confused me?